src/expandableCalendar/index.tsx

```diff
@@ -580,7 +583,7 @@ const ExpandableCalendar = (props: ExpandableCalendarProps) => {
         onDayPress={_onDayPress}
         onVisibleMonthsChange={onVisibleMonthsChange}
         pagingEnabled
-        scrollEnabled={isOpen}
+        scrollEnabled={isOpen && !disableMonthScroll}
```

Comment:
Since CalendarList props allow passing ScrollView's `scrollEnabled` I prefer to use this one instead of adding another prop to the API.
Let's try moving line 586 before the `{...others}` (line 579) to allow overriding the 'scrollEnabled' prop by passing it instead of 'disableMonthScroll' to the ExpandableCalendar component.

Reply:
Do you think it will confuse the user because we already had the `disableWeekScroll` prop ?  They will think that using the prop `scrollEnabled` will disable scrolling behavior for both states opened and closed, not only opened. For that, I think we should use another prop for clarity.
If you insist on making the change in your opinion please let me know.

Reply:
I understand your point, but in the case of 'disableWeekScroll' we change the component from WeekCalendar to a simple Week component. In your implementation you're not replacing the CalendarList with a simple Calendar component (which is an option), but only passing it to the CalendarList's FlatList implementation.

Reply:
I tried to do the same logic as the WeekCalendar but seems using the normal Calendar component it will break. So in this case must we use the `scrollEnabled` ?

<img src="https://github.com/user-attachments/assets/c78a50ed-0274-4f41-ad29-44d0f48f0611" width="400">

Reply:
any update @Inbal-Tish ?